src/client/activation/jedi/languageServerProxy.ts

```diff
@@ -120,8 +125,10 @@ export class JediLanguageServerProxy implements ILanguageServerProxy {
         }
     }
 
-    // tslint:disable-next-line: no-empty
-    public loadExtension(_args?: {}) {}
+    // eslint-disable-next-line class-methods-use-this
+    public loadExtension(/* _args?: {} */): void {
```

Comment:
Why not remove the arguments entirely?

```suggestion
    public loadExtension(): void {
```

Reply:
Because if someone actually does want to use them there's a little less thinking behind what the parameter might be. If they weren't already there I wouldn't care, but this seemed easy enough.

Reply:
If somebody wants to use them, they can (and probably should) look a the interface being implemented, I don't think keeping the arguments around is necessary.

Reply:
Definitely not necessary; this comes down to laziness. 😁 

Anyway, I have removed the comments.